### ISSUE TITLE
iOS: image alt text editor missing

### ISSUE THREAD

**Issue by ingernet** [tooot-app/app#683]
iOS version: 16.2
Tooot version: 4.8.5

I can no longer add alt text to an image in a post. I've tried it with single image posts and multi-image posts. I've tried to add alt text upon initial post and upon editing the post.

The image alt text tool has just gone poof.

![DD785DA4-9865-4B9B-A648-26B4DF7ECF5D](https://user-images.githubusercontent.com/6075258/215339014-ca53c43f-b64b-49a6-b355-a7efbfd16942.jpeg)
